Airports/EGDI/Sector.txt

```diff
@@ -0,0 +1,7 @@
+CIRCLE_SECTORLINE:DITWR:EGDI:2
+DISPLAY:DITWR:DITWR:DIAPP
+DISPLAY:DIAPP:DITWR:DIAPP
+
+CIRCLE_SECTORLINE:DIAPP MATZ CIRCLE:EGDI:3
+DISPLAY:Swanwick Mil South West:Swanwick Mil South West:DIAPP
```

Comment:
Given that the APP role for Merryfield is always covered by Yeovil, do you want to display a ring around both Yeovil and Merryfield to Swanwick Mil, or just around Yeovil?